File: src/me/riseremi/utils/Shift.java
package me.riseremi.utils;

/**
 *
 * @author riseremi <riseremi at icloud.com>
 */
public class Shift {

    public static int ShiftNorth(int p, int distance) {
        return (p - distance);
    }

    public static int ShiftSouth(int p, int distance) {
        return (p + distance);
    }

    public static int ShiftEast(int p, int distance) {
        return (p + distance);
    }

    public static int ShiftWest(int p, int distance) {
        return (p - distance);
    }

}

File: src/me/riseremi/utils/Shift.kt
package me.riseremi.utils

/**
 *
 * @author riseremi <riseremi at icloud.com>
</riseremi> */
class Shift {
    companion object {
        fun ShiftNorth(p: Int, distance: Int): Int {
            return p - distance
        }

        fun ShiftSouth(p: Int, distance: Int): Int {
            return p + distance
        }

        fun ShiftEast(p: Int, distance: Int): Int {
            return p + distance
        }

        fun ShiftWest(p: Int, distance: Int): Int {
            return p - distance
        }
    }

}
